meta-resin-common/recipes-core/initrdscripts/files/machineid

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+machineid_enabled() {
+    mount -t ext4 /dev/disk/by-label/resin-state $ROOTFS_DIR/mnt/state
```

Comment:
But this variable is a constant through the framework, I do not see the need for quotes here.